django/contrib/admin/options.py

```diff
@@ -109,6 +109,7 @@ class BaseModelAdmin(six.with_metaclass(forms.MediaDefiningClass)):
     formfield_overrides = {}
     readonly_fields = ()
     ordering = None
+    orderable_by = ()
```

Comment:
I think the idea is `sortable_by = None` is the default case of all fields sortable and `sortable_by = ()` means no fields are sortable.